packages/react-devtools-shared/src/node_modules/react-window/src/FixedSizeList.js

```diff
@@ -1,10 +1,12 @@
 // @flow
 
+import type {ComponentType} from "react";
+
 import createListComponent from './createListComponent';
 
 import type { Props, ScrollToAlign } from './createListComponent';
 
-const FixedSizeList = createListComponent({
```

Comment:
This folder is vendored. We shouldn't be changing it?

Reply:
I guess we already changed it once before. Not sure why it's vendored.

Reply:
Yeah, not sure what's going on here. Maybe it should just be installed as a regular npm dependency? I suspect @bvaughn forked something in there or it was a pre-release at some point.

Reply:
No idea. Looking back through the history, I found 6ff733873c662086f919baced2bb80fcffc1b343 which is useless (yay me). Sorry 😄

We'd have to diff source between the vendored and non-vendored one to see what I changed.